lib/tasks/encrypt.rake

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+namespace :encrypt do
+  desc "Migrate user info to new encryption format"
+  task :user => :environment do
+    updated_count = 0
+    error_count = 0
+
+    User.find_each do |user|
+      user.token = user.token_old
+
+      if user.save
+        updated_count += 1
+      else
+        puts "** Error while updating ID: #{user.id}"
+        error_count += 1
+      end
+    end
+
+    puts "Update complete. Total rows in table: #{User.count}"
+    puts "Updated #{updated_count} record(s). Hit #{error_count} error(s)."
```

Comment:
I wonder if we want to wrap this inside a transaction and fail the whole task in case of errors? so users do not end up with a half-converted table

Reply:
Does it matter?  If it fails halfway through you can just run it again and it will re-assign ones that have been already assigned to the same thing.  🤷‍♀

Reply:
Is it expected that rerunning will just work? if so I do not see a problem, but if there is some action needed to get it to run a half-broken table could be a problem. Maybe I am not sure what the failure modes are, the fact that there is an error count made me think there could be user interaction necessary to fix those

Reply:
> Is it expected that rerunning will just work?

I think so.  If there are issues with individual records, you would have to fix those, but then you should be able to just run it again.  Not sure if I'm missing something?

In any case, having a transaction doesn't hurt, so I've added it.  :)